kind.sh

```diff
@@ -141,16 +141,24 @@ parse_command_line() {
 
 install_kind() {
     echo 'Installing kind...'
-    curl -sSLo kind "https://github.com/kubernetes-sigs/kind/releases/download/$version/kind-linux-amd64"
-    chmod +x kind
-    sudo mv kind /usr/local/bin/kind
+    if [ "$RUNNER_OS" == "macOS" ]; then
+        brew install kind
+    else
+        curl -sSLo kind "https://github.com/kubernetes-sigs/kind/releases/download/$version/kind-linux-amd64"
+        chmod +x kind
+        sudo mv kind /usr/local/bin/kind
+    fi
 }
 
 install_kubectl() {
     echo 'Installing kubectl...'
-    curl -sSLO "https://storage.googleapis.com/kubernetes-release/release/$kubectl_version/bin/linux/amd64/kubectl"
-    chmod +x kubectl
-    sudo mv kubectl /usr/local/bin/kubectl
+    if [ "$RUNNER_OS" == "macOS" ]; then
+        brew install kubectl
```

Comment:
I think this change the behavior of the action as it would install whatever kubectl version is defined in brew regardless of the kind version that is set in the action.